packages/itinerary-body/src/ItineraryBody/accessibility-rating.tsx

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import styled from "styled-components";
+import { Wheelchair } from "@styled-icons/foundation/Wheelchair";
+
+interface WrapperProps {
+  color: string;
+  large: boolean;
+}
+
+const Wrapper = styled.div<WrapperProps>`
+  align-items: center;
+  background-color: ${props => props.color};
+  border-radius: ${props => (props.large ? "4px" : "20px")};
+  display: flex;
+  justify-content: space-between;
+  margin-top: 0.25em;
+  max-height: 50px;
+  padding: 0.25em 0.6em 0.25em 0.4em;
+  width: ${props => (props.large ? "65px" : "50px")};
+  word-wrap: anywhere; /* this can often look quite bad, but helps encourage icons */
+`;
+const StatusWrapper = styled.span`
+  flex: 1;
+
+  span {
+    display: block;
+  }
+`;
+
+const TextWrapper = styled.span`
+  padding-top: 3px;
+  font-weight: 600;
+`;
+
+/**
+ * Component which renders a label with a color and icon depending on
+ * a given accessibility score. The color and icon are set by a given gradation map.
+ */
+const AccessibilityRating = ({
+  gradationMap,
+  grayscale = false,
+  large = false,
+  score
+}: {
+  gradationMap?: Record<
+    number,
+    { color: string; icon?: JSX.Element; text?: string }
+  >;
+  grayscale: boolean;
+  large: boolean;
+  score: number;
+}): JSX.Element => {
```

Comment:
In package `location-icon`, we first started using `ReactElement` as return type, so for consistency we should return `ReactElement` here too.

Reply:
Sounds good! Implemented in 9b052113f918c4d8c638f99fb623ad71b8bba15d